inst/components/scss/sidebar.scss

```diff
@@ -54,14 +64,16 @@ $bslib-sidebar-column-sidebar: Min(calc(100% - var(--bslib-sidebar-icon-size)),
     border-right: var(--bslib-sidebar-vert-border);
     border-top-right-radius: 0;
     border-bottom-right-radius: 0;
-    background-color: var(--bslib-sidebar-bg);
     color: var(--bslib-sidebar-fg);
+    background-color: var(--bslib-sidebar-bg);
+    backdrop-filter: blur(5px);
 
     > .sidebar-content {
       display: flex;
       flex-direction: column;
       gap: var(--bslib-spacer, 1rem);
       padding: var(--bslib-sidebar-padding);
+      padding-top: calc(var(--bslib-sidebar-padding, 1rem) * 2);
```

Comment:
Explain that this extra padding is for the toggle and that we undo it when there is no toggle